docs/getting-started.md

```diff
@@ -69,33 +64,37 @@ the assets needed for MDLv2. Put the following within `index.html` in the `greet
 </html>
 ```
 
-To view the page, run `live-server` within the `greeting-app` directory. This will open up your
+View this page by running `live-server` within the `greeting-app` directory. This will open up your
 browser to the URL which is serving our `index.html` file. You can leave `live-server` running for
 the duration of this guide.
 
+```
+cd greeting-app
+live-server
+```
+
 Let's take a look at a few aspects of the above HTML.
 
 * **No JavaScript necessary (yet)** - Because we aren't using any dynamic components, we only need
   to include the MDLv2 CSS, so that we can apply the proper CSS classes to our elements. With MDLv2,
   JavaScript is only necessary for dynamic components whose UI needs to be made aware of events
   happening on the page. As we develop our greeting app, we'll
   add in the necessary JavaScript.
-* **No automatic DOM rendering** - For all components, MDL does not render _any_ DOM elements
+* **No automatic DOM rendering** - For all components, MDLv2 does not render _any_ DOM elements
```

Comment:
`MDLv2` to just `MDL`. V2 is an assignment version not the actual name of the project.